fe/src/components/searchBar/Seachbar.js

```diff
@@ -0,0 +1,65 @@
+import './searchbar.scss';
+import mock from "./mock.json";
+import React, { useContext, useEffect, useState } from 'react'
+import { getTablesEndpoint } from '../../util/Endpoints'
+import { MapContext } from '../../contexts/mapContext'
+import { YearContext } from '../../contexts/yearContext'
+import { faWindowClose } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { IconContext } from 'react-icons/lib';
+
+export default function Searchbar() {
+  
+  const[searchTerm, setSearchTerm] = useState('');
+  const { mapContext , setMapContext} = useContext(MapContext);
+  const { yearData, setYearData } = useContext(YearContext);
+  const [hasFocus, setFocus] = useState(false);
+  var viewCompanyList;
+
+  useEffect(() => {
+
+    async function getTablesAsyncWrapper() {
+        const res = await getTablesEndpoint(yearData.selected)
+        console.log("TABLES: ", yearData)
+        setMapContext(res)
+        
+    }
+    console.log("MAP CONTEXT: ", yearData)
+        if (yearData && yearData.selected) {
+            getTablesAsyncWrapper();
+        }
+    }, [yearData])
+        
+  if (hasFocus) {
```

Comment:
should wrap this in a useEffect() hook, (like above), and put searchTerm, and hasFocus in the [arguments] (at the bottom of the hook).

Merge errors here should be straight forward